syncoid

```diff
@@ -452,13 +452,13 @@ sub checkcommands {
 
 	if ($avail{'sourcecompress'} eq '') {
 		if ($compressargs{'rawcmd'} ne '') {
-			print "WARN: $compressargs{'rawcmd'} not available on source $s- sync will continue without compression.\n";
+			if (!$quiet) { print "WARN: $compressargs{'rawcmd'} not available on source $s- sync will continue without compression.\n"; }
```

Comment:
I personally would not surpres warnings, I would really love to see those in my mailbox

Reply:
Agreed - hence the separate commit - I wasn't sure how the maintainer(s) feel.

Going by the help however, this is probably technically correct:
```
  --quiet               Suppresses non-error output
```

---

I've seen the "_follow behaviour of other tools_" mentality on this project - anyone able/willing to compare with behaviour of `rsync`'s `--quiet` flag?
```
  -q, --quiet                 suppress non-error messages
```